packages/renderer/src/lib/ContainerList.svelte

```diff
@@ -347,6 +371,35 @@ function errorCallback(container: ContainerInfoUI, errorMessage: string): void {
   container.state = 'ERROR';
   containerGroups = [...containerGroups];
 }
+
+function iconClass(container: ContainerInfoUI): string | undefined {
```

Comment:
it looks like this method will be called only if we refresh container list
but AFAIK icon should be also refreshed when contexts are being updated no ?